command/cp.go

```diff
@@ -107,6 +107,16 @@ var copyCommandFlags = []cli.Flag{
 		Value:   defaultPartSize,
 		Usage:   "size of each part transferred between host and remote server, in MiB",
 	},
+	&cli.StringFlag{
+		Name:    "sse-encrypt",
+		Aliases: []string{"sse"},
+		Usage:   "server side encryption",
+	},
+	&cli.StringFlag{
+		Name:    "key-id",
+		Aliases: []string{"kid"},
+		Usage:   "encryption key id",
```

Comment:
since in the future server side encryptions other than kms might be supported, i thought using key-id might be reusable in those cases. what do you think?

Reply:
since in the future other encryption methods might be supported, key-id parameter can be reused in those cases. what do you think?